src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -124,6 +124,11 @@
   {:name schema/Str
    :files [EnvironmentClassesFileEntry]})
 
+(def EnvironmentModulesInfo
+  "Schema for the return payload an environment_classes request."
+  {:name schema/Str
+   :modules schema/Any}) ; TODO: fixme
```

Comment:
@camlow325 yeah, I'll have to make it that. I also still need to confirm what shows up in the Puppet::Modules data structure if it is a module not installed from forge.